### ISSUE TITLE
[Bug]: Slider track is hard to click

### ISSUE THREAD

**Issue by brunnerh** [carbon-design-system/carbon#9646]
### Package

carbon-components-react

### Browser

Chrome, Firefox

### Package version

Checked on Carbon website with:
Vanilla Components version ^10.43.0
React Components version ^7.43.0

### Description

The track of the slider has a two pixel high hitbox that can be clicked (equal to visual representation of track).
The clickable area should be much higher than that; currently it is almost impossible to click without spending additional time to precisely positioning the cursor.

### CodeSandbox example

https://codesandbox.io/s/slider-track-hitbox-issue-4wf7o

### Steps to reproduce

- Create Slider
- Try to click on track (not thumb)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
